charts/athens-proxy/templates/deployment.yaml

```diff
@@ -8,22 +8,47 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 spec:
-  
   replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+      chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+      release: "{{ .Release.Name }}"
   template:
     metadata:
       labels:
         app: {{ template "fullname" . }}
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/config-gocenter.yaml") . | sha256sum }}
+        checksum/config: {{ include (print $.Template.BasePath "/config-ssh-git-servers.yaml") . | sha256sum }}
```

Comment:
This maybe will only work for `config-ssh-git-servers.yaml`, not for `config-gocenter.yaml` as the two annotation names are the same.

Reply:
it will work for both, as files listed there have different names

Reply:
`helm install athens-proxy/ -n athens --dry-run --debug`
```
...
  template:
    metadata:
      labels:
        app: athens-athens-proxy
        chart: "athens-proxy-0.2.1"
        release: "athens"
      annotations:
        checksum/config: c1c40d75065587c49691755b457d72d8f188c8762ef24b61d45c476393a7d90c
        checksum/config: 95356df6ef3f8b1b87e99b54f0d42bc918c839a0681ab705c79125d3abfbcfdd
        checksum/secret: 51c479514eb71530bfe8f6e4777fd957ad89413993b0804256d20b7521cc1739
...
```
`gocenter` and `sshGitServers` enabled there, and all checksum are different